InstallFullDB.sh

```diff
@@ -270,194 +918,1196 @@ then
+  if ! apply_world_db_core_update;then
+    false
+    return
+  fi
+  if ! apply_char_db_core_update;then
+    false
+    return
+  fi
+  if ! apply_realm_db_core_update;then
+    false
+    return
+  fi
+  if ! apply_logs_db_core_update;then
+    false
+    return
+  fi
+  true
+}
 
-#
-#               CMaNGOS custom updates file
-#
-# Apply cmangos_custom.sql
-echo "> Trying to apply ${ADDITIONAL_PATH}utilities/cmangos_custom.sql ..."
-$MYSQL_COMMAND < ${ADDITIONAL_PATH}utilities/cmangos_custom.sql
-if [[ $? != 0 ]]
-then
-  echo "ERROR: cannot apply ${ADDITIONAL_PATH}utilities/cmangos_custom.sql"
-  exit 1
-fi
-echo "  CMaNGOS custom updates successfully applied"
-echo
-echo
+function apply_ahbot_data()
+{
+  # Apply optional AHBot commands documentation
+  if [ "$AHBOT" == "YES" ]; then
+    echo "> Trying to apply $CORE_PATH/sql/base/ahbot ..."
+    for f in "$CORE_PATH/sql/base/ahbot/"*.sql
+    do
+        if ! execute_sql_file "$WORLD_DB_NAME" "$f" "  - Applying $f"; then
+          false
+          return
+        fi
+    done
+    echo
+  fi
+  true
+}
 
-#
-#    LOCALES
-#
-if [ "$LOCALES" == "YES" ]
-then
-  echo "> Trying to apply locales data..."
-  for UPDATEFILE in ${ADDITIONAL_PATH}locales/*.sql
+# Apply dbc folder
+function apply_full_dbc_data()
+{
+  echo "> Trying to apply dbc datas"
+  for f in "$CORE_PATH/sql/base/dbc/original_data/"*.sql
   do
-    if [ -e "$UPDATEFILE" ]
-    then
+    local fName=$(basename "$f")
+    if ! execute_sql_file "$WORLD_DB_NAME" "$f" "  - Applying $fName"; then
+      false
+      return
+    fi
+  done
+  echo "  Original DBC data were applyed successfully!"
+  echo
+
+    # Apply dbc changes (specific fixes to known wrong/missing data)
+  echo "> Trying to apply CMaNGOS fixes to dbc datas ... "
+  for f in "$CORE_PATH/sql/base/dbc/cmangos_fixes/"*.sql;do
+    local fName=$(basename "$f")
+    if ! execute_sql_file "$WORLD_DB_NAME" "$f" "  - Applying $fName"; then
+      false
+      return
+    fi
+  done
+  echo "  CMaNGOS fixes for DBC data were applyed successfully!"
+  echo
+  true
+}
+
+# Apply scriptdev2.sql
+function apply_full_scriptdev2_data()
+{
+  if ! execute_sql_file "$WORLD_DB_NAME" "$CORE_PATH/sql/scriptdev2/scriptdev2.sql" "> Trying to apply ScripDev2 data"; then
+    false
+    return
+  fi
+  echo
+  true
+}
+
+# Apply full ACID file
+function apply_acid_data()
+{
+  if ! execute_sql_file "$WORLD_DB_NAME" "ACID/acid_${EXPENSION,,}.sql" "> Trying to apply ACID file"; then
+    false
+    return
+  fi
+  echo
+  true
+}
+
+# Apply cmangos_custom.sql
+function apply_custom_data()
+{
+  if ! execute_sql_file "$WORLD_DB_NAME" "${ADDITIONAL_PATH}utilities/cmangos_custom.sql" "> Trying to apply cmangos_custom.sql"; then
+    false
+    return
+  fi
+  echo
+  true
+}
+
+# Apply locales
+function apply_locales()
+{
+  if [ "$LOCALES" == "YES" ]; then
+    echo "> Trying to apply locales data (May take some minutes) ..."
+    for UPDATEFILE in ${ADDITIONAL_PATH}locales/*.sql
```

Comment:
I received error like this when I try to use additional option like Locales:

```> Trying to apply locales data (May take some minutes) ...
  - Applying locales/BroadcastTextLocales.sql ... SUCCESS
  - Applying locales/OtherLocales.sql ... SUCCESS
  - Applying locales/BroadcastTextLocales.sql ... FAILED!
>>> ERROR 1062 (23000) at line 9: Duplicate entry '14746-deDE' for key 'broadcast_text_locale.PRIMARY'
Press space to continue...
```
I think it's just looping twice through the files in locales folder.